### DISCUSSION TITLE
httpx github action go  question

### DISCUSSION THREAD

**Discussion by jareddarkweb** [projectdiscovery/httpx#1904] [Q&A]
I get an error when using httpx github action..

this is the error 

[httpx-scan](https://github.com/jareddarkweb/xxxxx/actions/runs/xxxxxx/job/xxxx#logs)
failed 11 hours ago in 9s
Search logs
2s
Current runner version: '2.3[1](https://github.com/jareddarkweb/xxxx/actions/runs/xxxx/job/29542477903#step:1:1)9.1'
Operating System
Runner Image
Runner Image Provisioner
GITHUB_TOKEN Permissions
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff[2](https://github.com/jareddarkweb/xxx/actions/runs/10659649950/job/29542477903#step:1:2)bd294095638e18286ca9a3d195[6](https://github.com/jareddarkweb/Rxxxactions/runs/10659649950/job/29542477903#step:1:7)744)
Download action repository 'actions/setup-go@v3' (SHA:xxx87c207a568)
Download action repository 'projectdiscovery/httpx-action@main' (SHA:dbxxxfbe8b[30](https://github.com/jareddarkweb/Recon002/actions/runs/10659649950/job/29542477903#step:1:34)5b9a532b22635b)
Download action repository 'actions/upload-artifact@v2' (SHA:82c141cc518b40xxx8e7aafc9c2fa2)
Complete job name: httpx-scan
0s
4s
0s
Run projectdiscovery/httpx-action@main
Run [ ! -x /home/runner/go/bin/httpx ] && go install -v github.com/projectdiscovery/httpx/cmd/httpx@latest
can't load package: package github.com/projectdiscovery/httpx/cmd/httpx@latest: cannot find package "github.com/projectdiscovery/httpx/cmd/httpx@latest" in any of:
	/opt/hostedtoolcache/go/1.2.2/x64/src/pkg/github.com/projectdiscovery/httpx/cmd/httpx@latest (from $GOROOT)
	($GOPATH not set)
Error: Process completed with exit code [1](https://github.com/jareddarkweb/Recon002/actions/runs/10659649950/job/29542477903#step:4:1).
0s
0s
0s

IS this a problem on httpx end? i am but confused how to fix it , and would love to httpx with github actions

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
